ios/RCTTWVideoModule.m

```diff
@@ -311,12 +318,27 @@ -(NSMutableDictionary*)convertLocalVideoTrackStats:(TVILocalVideoTrackStats *)st
       builder.audioTracks = @[self.localAudioTrack];
     }
 
+
+    self.localDataTrack = [TVILocalDataTrack track];
+
+    if (self.localDataTrack) {
+      builder.dataTracks = @[self.localDataTrack];
+    }
+
     builder.roomName = roomName;
   }];
 
   self.room = [TwilioVideo connectWithOptions:connectOptions delegate:self];
 }
 
+RCT_EXPORT_METHOD(sendString:(nonnull NSString *)message) {
+    NSLog(@"Send String!");
```

Comment:
Some debugging stuff could be removed =).

Reply:
done